app/controllers/application_controller.rb

```diff
@@ -9,12 +9,21 @@ class ApplicationController < ActionController::API
 
   before_action :authenticate
   before_action :set_app_info_headers
-  skip_before_action :authenticate, only: [:cors_preflight]
+  skip_before_action :authenticate, only: [:cors_preflight, :routing_error]
 
   def cors_preflight
     head(:ok)
   end
 
+  def routing_error
+    raise Common::Exceptions::RoutingError.new(params[:path])
+  end
+
+  def action_missing(m, *args, &block)
```

Comment:
Incidentally, it would raise ActionController::ActionNotFound or something like that